ext/store/bigquery/bigquery_test.go

```diff
@@ -934,6 +934,7 @@ func TestBigqueryStore(t *testing.T) {
 			client.On("TableHandleFrom", mock.Anything, mock.Anything).Return(handle).Maybe()
 			client.On("ExternalTableHandleFrom", mock.Anything, mock.Anything).Return(handle).Maybe()
 			client.On("ViewHandleFrom", mock.Anything, mock.Anything).Return(handle).Maybe()
+			client.On("RoutineHandleFrom", mock.Anything, mock.Anything).Return(handle).Maybe()
```

Comment:
please create a separate unit test, or change to test table, since the way it is being configured does not really capture the behavior properly

Reply:
updated